contrib/debian/manpages/bitcoin-cli.1

```diff
@@ -0,0 +1,52 @@
+.TH BITCOIN-CLI "1" "February 2015" "bitcoin-cli 0.10" 
+.SH NAME
+bitcoin-cli \- the remote procedure call client for bitcoin core. 
+.SH SYNOPSIS
+bitcoin-cli [options] <command> [params] \- Send command to Bitcoin Core. 
+.TP
+bitcoin-cli [options] help \- List Bitcoin Core commands.
```

Comment:
Maybe: Ask Bitcoin Core for a list of supported commands (to indicate that the result of this command depends on the server, and not the client).